appmgr/src/update/mod.rs

```diff
@@ -209,7 +213,24 @@ async fn maybe_do_update(ctx: RpcContext) -> Result<Option<Arc<Revision>>, Error
                         None,
                     )
                     .await
-                    .expect("")
+                    .expect("");
+                // TODO: refactor sound lib to make compound tempos easier to deal with
+                UPDATE_FAILED_1
+                    .play()
+                    .await
+                    .expect("could not play song: update failed 1");
+                UPDATE_FAILED_2
+                    .play()
+                    .await
+                    .expect("could not play song: update failed 2");
+                UPDATE_FAILED_3
+                    .play()
+                    .await
+                    .expect("could not play song: update failed 3");
+                UPDATE_FAILED_4
+                    .play()
+                    .await
+                    .expect("could not play song: update failed 4");
```

Comment:
why are we playing 4 songs here?

Reply:
the song abstraction doesn't handle multitempo well